app/javascript/mastodon/features/emoji/emoji_mart_data_light.ts

```diff
@@ -30,10 +30,8 @@ const emojis: Emojis = {};
 // decompress
 Object.keys(shortCodesToEmojiData).forEach((shortCode) => {
   const [_filenameData, searchData] = shortCodesToEmojiData[shortCode];
-  const native = searchData[0];
-  let short_names = searchData[1];
-  const search = searchData[2];
-  let unified = searchData[3];
+  const [native, _short_names, search, _unified] = searchData;
+  let [_native, short_names, _search, unified] = searchData; // eslint-disable-line prefer-const
```

Comment:
Rather than disabling `eslint`, for this line, you can rewrite this more compactly like this:

```ts
  const [native, short_names, search, unified] = searchData;

  emojis[shortCode] = {
    native,
    search,
    short_names: short_names ? [shortCode].concat(short_names) : undefined,
    unified: unified ?? unicodeToUnifiedName(native),
  };
```

What do you think?

Reply:
Thanks for your feedback.
Indeed, your code is much simpler.
I've updated the code accordingly.